src/edu/cmu/cs/sasylf/Main.java

```diff
@@ -18,173 +10,242 @@
+      if (args[i].equals("--compwhere")) {
+        edu.cmu.cs.sasylf.util.Util.COMP_WHERE = true;
+        continue;
+      }
+      if (args[i].equals("--LF")) {
+        edu.cmu.cs.sasylf.util.Util.EXTRA_ERROR_INFO = true;
+        continue;
+      }
+      if (args[i].equals("--task")) {
+        edu.cmu.cs.sasylf.util.Util.SHOW_TASK_COMMENTS = true;
+        DSLToolkitParser.addListener(TaskReport.commentListener);
+        continue;
+      }
+      if (args[i].equals("--verbose")) {
+        edu.cmu.cs.sasylf.util.Util.VERBOSE = true;
+        continue;
+      }
+      if (args[i].equals("--debug")) {
+        edu.cmu.cs.sasylf.util.Util.DEBUG = true;
+        continue;
+      }
+      if (args[i].equals("--waitForCR")) {
+        System.out.println("Press <ENTER> to continue");
+        while (System.in.read() != '\n') {
+          // do nothing
+        }
+        continue;
+      }
+      if (args[i].equals("--XContextIsSyntax")) {
+        edu.cmu.cs.sasylf.util.Util.X_CONTEXT_IS_SYNTAX = true;
+        continue;
+      }
+      if (args[i].startsWith("--root=")) {
+        dir = args[i].substring(7);
+        File root = new File(dir);
+        if (!root.isDirectory()) {
+          if (root.exists()) {
+            System.err.println("Not a directory: " + dir);
+            System.exit(-1);
+          } else {
+            System.err.println("No such file or directory: " + dir);
+            System.exit(-1);
+          }
+        }
+        mf = new RootModuleFinder(root);
+        continue;
+      }
+      if (args[i].startsWith("--path=")) {
+        mf = new PathModuleFinder(args[i].substring(7));
+        continue;
+      }
+      String filename = args[i];
+      File file = null;
+      ModuleId id = null;
+      Proof pf = null;
+      if (args[i].startsWith("--stdin"))
+        filename = "stdin";
+      if (mf != null) {
+        try {
+          id = new ModuleId(filename);
+        } catch (RuntimeException ex) {
+          System.err.println(ex.getMessage());
+          exitCode = -1;
+          continue;
+        }
+        try {
+          pf = mf.findProof(id, new Location("<commandline>", 0, 0));
+        } catch (SASyLFError e) {
+          // already handled
+        }
+      } else {
+        if (!args[i].startsWith("--stdin")) {
+          file = new File(filename);
+          if (!file.canRead()) {
+            System.err.println("Could not open file " + filename);
+            exitCode = -1;
+          }
+        }
+        try {
+          /**
+           * We take the input from stdin and turn that into a VSDocument that
+           * is needed for the quickfixes.
+           */
+          InputStreamReader r;
+          if (args[i].startsWith("--stdin"))
+            r = new InputStreamReader(System.in);
+          else
+            r = new InputStreamReader(new FileInputStream(file), "UTF-8");
+
+          BufferedReader reader = new BufferedReader(r);
+          StringBuilder sb = new StringBuilder();

```

Comment:
tabs for indentation has the advantage that the viewer can choose what the spacing will be.  You've changed everything to hard spaces.  That not only obscures whether you did anything (did you?) but also means one cannot choose a different indentation view.  Please revert these changes.

Reply:
Is there a style guide I could reference for the project? I will configure my formatter to follow it. This should solve most of the issues with white spaces, import orders, etc.